traffic/v2/index.ts

```diff
@@ -3,7 +3,7 @@ import type { APITrafficServerGameName } from './server';
 export * from './server';
 export * from './traffic';
 
-export const APITrafficVersion = '2';
+export const APITrafficVersion = '2' as const;
```

Comment:
This change should be done to the website API, too, or should be reverted here.

https://github.com/TruckersMP/API-Types/blob/1e77acc152a85e81c15c6e00966d7e5c140020bb/web/v2/index.ts#L9